src/components/Attachments/AttachmentView/AttachmentViewVideo/index.js

```diff
@@ -0,0 +1,27 @@
+import React, {memo} from 'react';
+import VideoPlayer from '@components/VideoPlayer';
+import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import compose from '@libs/compose';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+function AttachmentViewVideo({source}) {
+    const {isSmallScreenWidth} = useWindowDimensions();
+    const shouldUseSharedElementTransition = !isSmallScreenWidth;
+
+    return (
+        <VideoPlayer
+            url={source}
+            shouldPlay={false}
+            shouldUseSharedVideoElement={shouldUseSharedElementTransition}
+        />
+    );
+}
+
+AttachmentViewVideo.propTypes = propTypes;
+AttachmentViewVideo.defaultProps = {};
```

Comment:
Do we need this, if we pass an empty object?

Reply:
Ok, I've checked it, and we have a similar approach in a few places. So we can leave it as it is.